### ISSUE TITLE
Nested Scroll not working with Container.

### ISSUE THREAD

**Issue by harshbhavsar011** [eneim/toro#356]
Toro version : v3

- CoordinatorLayout     
     - AppBar Layout
            - CollpasableLayout
      - FrameLayout  ->  Nested scrollview -> Toro container 
- /CoordinatorLayout

 I have a fragment that has 1 frame-layout container in bottom . I am replacing 3 framgments in that frame container (FrameLayout) . Nested scroll working with default recycleview but not working with toro container .

**Comment by eneim** [comment]
@harshbhavsar011 Why do you need ``Nested scrollview -> Toro container`` ? 

As far as I know, this is not really good in term of performance.
To make sure it is a bug, can you debug your app, and make sure that ``Container#onScrollChanged(int)`` is called or not being called?

Last but not least, I suggest to consider to remove the nested scroll view if it is not necessary :D.

**Comment by harshbhavsar011** [comment]
Have a look into this video.

**Comment by eneim** [comment]
@harshbhavsar011 I see. How do you set the "app:behavior" for now? Can you share the xml with me? Email me if it need to be private --> nam@ene.im

**Comment by harshbhavsar011** [comment]
```

<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/main_content"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <android.support.design.widget.AppBarLayout
        android:id="@+id/appbar"
        android:layout_width="match_parent"
        android:layout_height="wrap_content">

        <android.support.design.widget.CollapsingToolbarLayout
            android:id="@+id/collapsing_toolbar"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            app:layout_scrollFlags="scroll">


            <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:background="@color/white"
                android:gravity="center"
                android:orientation="vertical"
                >

<!-- My Upper Profile View Content--> 

         </LinearLayout>
    </android.support.design.widget.CollapsingToolbarLayout>
 </android.support.design.widget.AppBarLayout>

 <FrameLayout
        android:id="@+id/frameContainer"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:nestedScrollingEnabled="false"
        app:layout_behavior="@string/appbar_scrolling_view_behavior" />


</android.support.design.widget.CoordinatorLayout>

```

**fragment_media** 

  `
  <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:orientation="vertical">

  

        <com.inheritx.flair.customeviews.EmptyRecyclerView
            android:id="@+id/rvData"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:scrollbars="none"/>


        <TextView
            android:id="@+id/tvEmptyTextview"
            style="@style/TextMediumSizeMedium"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="Data not found"
            android:textColor="@color/gray"
            />

</RelativeLayout>`

**fragment_profile_feed**

`<?xml version="1.0" encoding="utf-8"?>

    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:app="http://schemas.android.com/apk/res-auto"
        xmlns:tools="http://schemas.android.com/tools"
        android:id="@+id/main_content"
        app:layout_behavior="@string/appbar_scrolling_view_behavior"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:focusableInTouchMode="true"
        >
            <im.ene.toro.widget.Container
                android:id="@+id/rvFeed"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:overScrollMode="never"
                android:scrollbars="none"

                />

    </LinearLayout>
`

**Comment by eneim** [comment]
How about putting the Container out of LinearLayout (and even remove the LinearLayout)?

**Comment by harshbhavsar011** [comment]
Already tried all that possible XML changes. Also working in `MEDIA` recycleview.  Do you think This issue  can be occurs because of customized `container` (Toro recycleview) ?

**Comment by eneim** [comment]
Hmm. I don’t remember touching anything with that behaviour. It should work just like any other RecyclerView. I may need some inside like which View is detected by the Behavior and etc. Did you try to change the tab order, making Container to be the first tab?

**Comment by harshbhavsar011** [comment]
Yes tried not working! I have to dig into touching behaviour now!

**Comment by eneim** [comment]
Hmm. will try this as soon as I get home.

**Comment by eneim** [comment]
I hope there is nothing use reflection to inject the behaviour, but I will take a look deep down inside :(.
